.golangci.reference.yml

```diff
@@ -163,16 +163,27 @@ linters-settings:
     include-go-root: true
 
     # A list of packages for the list type specified.
+    # Can accept both string prefixes and string glob patterns.
     # Default: []
     packages:
       - github.com/sirupsen/logrus
+      - allow/**/pkg
 
     # A list of packages for the list type specified.
     # Specify an error message to output when a denied package is used.
     # Default: []
     packages-with-error-message:
       - github.com/sirupsen/logrus: 'logging is allowed only by logutils.Log'
 
+    # Specify rules by which the linter ignores certain files for consideration.
+    # Can accept both string prefixes and string glob patterns.
+    # The ! character in front of the rule is a special character
+    # which signals that the linter should negate the rule.
+    # This allows for more precise control, but it is only available for glob patterns.
+    # Default: []
+    ignore-file-rules:
+      - "!**/*_test.go"
```

Comment:
can you add another example without `!`

Reply:
Done

Reply:
@ldez Are you going to use Squash Merge? If not, I can squash two commits to have clear history.

I've made changes in a separate commit to make review easier.

Reply:
We will squash, you don't need to handle that.